Source/WebKit/UIProcess/RemoteLayerTree/RemoteAcceleratedEffectStack.h

```diff
@@ -57,6 +59,8 @@ class RemoteAcceleratedEffectStack final : public WebCore::AcceleratedEffectStac
 
     WebCore::AcceleratedEffectValues computeValues(MonotonicTime now) const;
 
+    bool m_animatesOpacity { false };
```

Comment:
Why is opacity special? Will we haves to track this for transform, filters etc too?

Reply:
Yes, we will have three flags in total: one for opacity, one for transform-related properties and one for filter-related properties.

Reply:
It would be fine to have a single-bit `OptionSet<>` in the first PR.

Reply:
Sure, will make that change before landing.